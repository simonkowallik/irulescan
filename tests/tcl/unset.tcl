unset -nocomplain -- var_x var_y var_z
unset -- var_x var_y var_z

unset fail_100 var_y var_z

unset -nocomplain fail_200 var_y var_z