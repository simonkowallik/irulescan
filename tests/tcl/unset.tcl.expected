DANGEROUS: missing options terminator `--` permits argument injection at `fail_100` in `unset fail_100 var_y var_z`
DANGEROUS: missing options terminator `--` permits argument injection at `fail_200` in `unset -nocomplain fail_200 var_y var_z`

